Functions.Templates/Templates/BlobTrigger-Java/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "defaultFunctionName": "BlobTriggerJava",
+    "description": "$BlobTrigger_description",
+    "name": "Blob trigger",
+    "language": "Java",
+    "category": [
+        "$temp_category_core",
+        "$temp_category_dataProcessing"
+    ],
+    "categoryStyle": "blob",
+    "enabledInTryMode": true,
+    "userPrompt": [
+        "connection",
+        "path"
+    ]
+}
```

Comment:
Metadata.json is only used by portal. Please remove if you are not using it for any other purpose.